perf/ftdc_rollups.go

```diff
@@ -71,6 +71,11 @@ func CreatePerformanceStats(dx *ftdc.ChunkIterator) (*PerformanceStatistics, err
 					perfStats.timers.extractedDurations,
 					extractValues(convertToFloats(metric.Values), lastValue)...,
 				)
+				// In order to avoid memory panics, reject
+				// anything larger than 4GB.
```

Comment:
why did you decide on 4GB?

Reply:
To be safe, it was panicking around 5GB.

Reply:
Still, what about what we do with it downstream: ([copying and then sorting](https://github.com/evergreen-ci/cedar/blob/301d41222f3ccd6ac661a13f25395465a8222801/perf/rollup_factory.go#L278-L279)). 

- Wouldn't copying would put us over again.
- Is the sort in-place? Will it take forever on such a large file?

Reply:
Are you suggesting we do half? Say, 2 or 2.5GB?

Reply:
Maybe even that would be too much? 
But do we have a sense of what the typical size of these files is? Like, if we set it at 500MB, for instance, we be disabling other workloads that are currently successful?

Reply:
I think most are quite small, but I am not sure, there are so many of these tests running.

Reply:
Logging the final size and alerting when we get something too big are probably worthwhile: 
- Logging the size: it would be good to have stats on how big these files are.
- Alerting: we wouldn't have noticed this error if it wasn't for the jobs stopping up the queue. If we're going to just be erroring and moving on points will be missing from the trend charts and we won't know it.

Reply:
followed up "in person": we're going to open a separate PR for logging when this happens.